dart/test/sentry_client_test.dart

```diff
@@ -477,6 +478,47 @@ void main() {
   test("options can't be null", () {
     expect(() => SentryClient(null), throwsArgumentError);
   });
+
+  group('EventProcessors', () {
+    SentryOptions options;
+
+    setUp(() {
+      options = SentryOptions(dsn: fakeDsn);
+      options.addEventProcessor(
```

Comment:
can we add a test for the `beforeSend` callback as well? the beforeSend is the most common callback to mutate an event

Reply:
✅